app/transformation.py

```diff
@@ -18,6 +18,10 @@ def convert_pdf_to_cmyk(input_data):
             '-sDEVICE=pdfwrite',
             '-sColorConversionStrategy=CMYK',
             '-sSourceObjectICC=app/ghostscript/control.txt',
+            '-dBandBufferSpace=100000000'
+            '-dBufferSpace=100000000',
+            '-dMaxPatternBitmap=1000000',
+            '-c 100000000 setvmthreshold -f',
```

Comment:
can you put `-f` on a new line as it's a separate command line arg

in fact why did we not have it before? 🤔

Reply:
As per https://ghostscript.com/doc/7.07/Use.htm

Interprets following non-switch arguments as file names to be executed using the normal run command. Since this is the default behavior, -f is useful only for terminating the list of tokens for the -c switch.

It should be on the same line. As discussed I'll add to the PR comments.

Reply:
Interprets following non-switch arguments as file names to be executed using the normal run command. Since this is the default behavior, -f is useful only for terminating the list of tokens for the -c switch.